src/factories/series/Column.ts

```diff
@@ -0,0 +1,92 @@
+module n3Charts.Factory.Series {
```

Comment:
I thought once the options are parsed we create a `Series` instance (with all the sane properties)